test/unittest/conftest.py

```diff
@@ -117,247 +56,116 @@ def an_insert(float_tensors_32):
                     'x': float_tensor(x),
                     'y': y,
                     'z': float_tensor(z),
-                    'update': False,
                 }
             )
         )
-    return data
-
 
-@pytest.fixture()
-def an_update(float_tensors_32):
-    float_tensor = float_tensors_32.encoders['torch.float32[32]']
-    data = []
-    for i in range(10):
-        x = torch.randn(32)
-        y = int(random.random() > 0.5)
-        z = torch.randn(32)
-        data.append(
-            Document(
-                {
-                    'x': float_tensor(x),
-                    'y': y,
-                    'z': float_tensor(z),
-                    'update': True,
-                }
-            )
+    if data:
+        data_layer.execute(
+            Collection(collection_name).insert_many(data),
+            refresh=False,
         )
 
-    return data
-
 
-@pytest.fixture
-def vector_index_factory(a_model):
-    def _factory(db, identifier, **kwargs) -> VectorIndex:
-        db.add(
-            Listener(
-                select=Collection('documents').find(),
-                key='x',
-                model='linear_a',
-            )
-        )
-        db.add(
-            Listener(
-                select=Collection('documents').find(),
-                key='z',
-                model='linear_a',
+def add_encoders(data_layer: Datalayer):
+    for n in [8, 16, 32]:
+        data_layer.add(tensor(torch.float, shape=(n,)))
+    data_layer.add(pil_image)
```

Comment:
data_layer rename to db

because thats the norm we are following acorss the project
wdyt @blythed

Reply:
Great, I've made those changes.